### PR TITLE
GP-048 add models of geopoints and geolist, also controllers with basic functionallity

### PR DESCRIPTION
**Pull request by ShefardPT**

### CODE REVIEW

File: src/GeoPing.Api/Services/Helper.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace GeoPing.Api.Services
+{
+    public class Helper : IHelper
```
Comment (jokerosky) [inline src/GeoPing.Api/Services/Helper.cs]: Very bade name for the class, Helper is like 'Service' or 'Class' or 'Component' or 'Controller', what of the scope of it's help?

File: src/GeoPing.Api/Models/Entities/UserPoint.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace GeoPing.Api.Models.Entities
+{
+    public class UserPoint
```
Comment (jokerosky) [inline src/GeoPing.Api/Models/Entities/UserPoint.cs]: it should has explicit Id  in order to Entity Framework could track it!

Comment (ShefardPT) [reply src/GeoPing.Api/Models/Entities/UserPoint.cs]: done

File: src/GeoPing.Api/Services/Helper.cs (hunk 1 above)
Comment (ShefardPT) [reply src/GeoPing.Api/Services/Helper.cs]: done

**Pull request merged**